langchain/vectorstores/faiss.py

```diff
@@ -409,3 +434,23 @@ def load_local(cls, folder_path: str, embeddings: Embeddings) -> FAISS:
         with open(path / "index.pkl", "rb") as f:
             docstore, index_to_docstore_id = pickle.load(f)
         return cls(embeddings.embed_query, index, docstore, index_to_docstore_id)
+
+    def _get_similarity_score(self, score: float) -> float:
```

Comment:
what's this needed for

Reply:
Will clean up. Not needed after moving the boundary validation to the base class